### PR TITLE
PresignedGetObject API.

### PR DESCRIPTION
**Pull request by BigUstad**
Args, Functional Test. Docs.

### CODE REVIEW

**Review by Praveenrajmani — APPROVED** [review]
LGTM

**Comment by Praveenrajmani** [comment]
Can you fix the CI failure? @BigUstad

**Comment by BigUstad** [comment]
> Can you fix the CI failure? @BigUstad

Another PR has been created for that purpose. #[490](https://github.com/minio/minio-dotnet/pull/490)

File: Minio.Functional.Tests/FunctionalTest.cs [hunk 1]
```diff
@@ -2361,7 +2361,11 @@ internal async static Task PresignedGetObject_Test1(MinioClient minio)
                                                 objectName,
                                                 filestream, filestream.Length, null);
                 ObjectStat stats = await minio.StatObjectAsync(bucketName, objectName);
-                string presigned_url = await minio.PresignedGetObjectAsync(bucketName, objectName, expiresInt);
+                PresingedGetObjectArgs preArgs = new PresingedGetObjectArgs()
```
Comment (krisis) [inline Minio.Functional.Tests/FunctionalTest.cs:2364]: typo: PresignedGetObjectArgs has a typo in other places too
```suggestion
                PresignedGetObjectArgs preArgs = new PresignedGetObjectArgs()
```

**Review by krisis — COMMENTED** [review]
Overall the changes LGTM, except for the typo.

**Review by krisis — APPROVED** [review]

**Pull request merged**